scripts/prepare_release.sh

```diff
@@ -78,10 +79,18 @@ mvn versions:set versions:commit -DnewVersion=${NEXT_SNAPSHOT} -DgenerateBackupP
 # Commits this next snapshot version.
 git commit -am "${NEXT_SNAPSHOT}"
 
-# Pushes the tag and release branch to Github.
+# Push the tag and release branch to Github.
```

Comment:
@elharo Would you be willing to explain the change?

I think previous was a shorthand of "It pushes" (third person singular present), rather than imperative.

Reply:
Thank you for explanation.